### ISSUE TITLE
Error when submitting to registry

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#1101]
---

Author Name: **Lauren Walker** (Lauren Walker)
Original Redmine Issue: 6955, https://projects.ecoinformatics.org/ecoinfo/issues/6955
Original Date: 2016-01-28
Original Assignee: Shaun Walbridge

---

I get the following error when trying to submit metadata to the dev.nceas registry:  The docid is not valid since it is null or contians the white space(s).

Attached is a screenshot of the simple metadata I provided.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **ben leinfelder** (ben leinfelder)
Original Date: 2016-01-28T23:32:47Z

---

I updated the permissions on the /var/metacat/temporary directory where the registry writes some necessary temporary files during upload. This resolved the blank docid issue.

But others continue to report that there is a schema validation error when submitting the form. It seems to be intermittent in that I was able to successfully upload a metadata record without encountering the error, but Lauren (and Julien and Greg) were not able to.

```
cvc-complex-type.2.4.a: Invalid content was found starting with element 'metadataProvider'. 
One of '{associatedParty, pubDate, language, series, abstract, keywordSet, additionalInfo, intellectualRights, distribution, coverage, purpose, maintenance, contact}' is expected.

```

The registry seems to incorrectly order the metadataProvider and associatedParty elements. The metadataProvider element should come before any associated party elements, but in Lauren's case it was placed after. She reports that on a subsequent test, she was able to upload a record so it is not consistent.
